### ISSUE TITLE
Gelli 1.3.3 from FDroid fails to open

### ISSUE THREAD

**Issue by richardneish** [dkanada/gelli#218]
Gelli 1.3.2 works OK, connecting over WiFi to a JellyFin server on my LAN.  When I upgrade to 1.3.3 I get the error "Gelli keeps stopping".

I'm not sure how to get the logs or error report.

Samsung Galaxy S20
Android version 12
Build number SP1A.210812.016.G980FXXUEFVDB
Kernel version 4.19.87-24107789

**Comment by AlexKalopsia** [comment]
Same happening on Xperia 1 II, stopped working after the 1.3.3 update

**Comment by jensstein** [comment]
I get this error on 1.3.3. It seem to be fixed by commit 23029b128fa607a8ef06d4f60926a9adbed02e68.
```
07-17 21:17:56.221  5990  5990 E AndroidRuntime: FATAL EXCEPTION: main
07-17 21:17:56.221  5990  5990 E AndroidRuntime: Process: com.dkanada.gramophone, PID: 5990
07-17 21:17:56.221  5990  5990 E AndroidRuntime: java.lang.RuntimeException: Unable to create application com.dkanada.gramophone.App: java.lang.IllegalStateException: Room cannot verify the da
ta integrity. Looks like you've changed schema but forgot to update the version number. You can simply fix this by increasing the version number.
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6767)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at android.app.ActivityThread.access$1500(ActivityThread.java:256)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2091)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:106)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at android.os.Looper.loopOnce(Looper.java:201)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:288)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:7870)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
07-17 21:17:56.221  5990  5990 E AndroidRuntime: Caused by: java.lang.IllegalStateException: Room cannot verify the data integrity. Looks like you've changed schema but forgot to update the ve
rsion number. You can simply fix this by increasing the version number.
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at androidx.room.RoomOpenHelper.checkIdentity(RoomOpenHelper.java:7)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at androidx.room.RoomOpenHelper.onOpen(RoomOpenHelper.java:2)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.onOpen(FrameworkSQLiteOpenHelper.java:2)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at android.database.sqlite.SQLiteOpenHelper.getDatabaseLocked(SQLiteOpenHelper.java:427)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at android.database.sqlite.SQLiteOpenHelper.getWritableDatabase(SQLiteOpenHelper.java:316)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.getWritableSupportDatabase(FrameworkSQLiteOpenHelper.java:2)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper.getWritableDatabase(FrameworkSQLiteOpenHelper.java:1)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at androidx.room.RoomDatabase.inTransaction(RoomDatabase.java:1)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at androidx.room.RoomDatabase.assertNotSuspendingTransaction(RoomDatabase.java:1)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at com.dkanada.gramophone.database.UserDao_Impl.getUsers(UserDao_Impl.java:2)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at com.dkanada.gramophone.App.onCreate(App.java:5)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1223)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6762)
07-17 21:17:56.221  5990  5990 E AndroidRuntime:        ... 9 more
```

**Comment by dkanada** [comment]
This was initially caused by a missing database migration but is resolved in the next update.
